lib/sparse/src/common/sparse_vector.rs

```diff
@@ -207,6 +207,14 @@ impl SparseVector {
         debug_assert!(result.validate().is_ok());
         result
     }
+
+    /// Create [RemappedSparseVector] from this vector in a naive way. Only suitable for testing.
+    pub fn into_remapped_for_testing_purposes(self) -> RemappedSparseVector {
```

Comment:
If it's for test only, you may cover by `#[cfg(test)]` and remove `for_testing_purposes` from name

Reply:
Moved under `#[cfg(feature = "testing")]`.